### ISSUE TITLE
Multi-screen is frustrating

### ISSUE THREAD

**Issue by jflorian** [sddm/sddm#677]
Perhaps this is a theme issue more than one of sddm, however I have a 4x1 screen layout so the credentials dialog appears on each of the four screens, yet only one of those shows any response to keyboard entry.  Because of draconian corporate policy our password requirements prove to be better typing practice than security so being able to see each blip per character is helpful to recognize when a fat finger got two keys instead of one.  Problem is, I never know where to look for those blips.  I practically have to start typing gibberish to learn where the focus is, erase that, then type my password.  This is hardly optimal.

At present, I'm using the 02-fedora theme (from Fedora 24).  I landed on this one because it's essential to have a username field -- our identities reside in LDAP and any theme with a user chooser fails to show them.

Is there a theme having, or an easy way to modify an existing theme to have, a single dialog, regardless of how many screens are present?

**Comment by plfiorini** [comment]
Both, there is a primaryScreen boolean hint available to themes but none of them use it due to problems with clone mode. Once those problems are resolved we can spread the word of primaryScreen.

**Comment by jflorian** [comment]
Okay, thanks for the info.  I look forward to seeing this land.

**Comment by piomiq** [comment]
> Both, there is a primaryScreen boolean hint available to themes but none of them use it due to problems with clone mode. Once those problems are resolved we can spread the word of primaryScreen.

Is that problem solved or not?
I have problem with multi monitor setup. I reported it some time ago here: #1219
I applied your suggestion with duplicate of section "Loader" and putting in one property 
`active: primaryScreen` and in second `visible: !primaryScreen`, but this doesn't work.
[Main.qml.txt](https://github.com/sddm/sddm/files/4037289/Main.qml.txt)
